tests/test_apply_variants.py

```diff
@@ -584,13 +584,13 @@ def test_reference_alignment_other_anchor_cases(self):
         reference_alignment = apply_variants(genome37.dna, variants, interval, reference_alignment=True)[1]
         self.assertEqual(reference_alignment, [0, 1, 2, 3, 4, (5, 0), (5, 1), 5, 6, 7])
 
-        # Test anchor right of deletion
+        # Test anchor middle of deletion
         variants = [Variant.from_string("chr1:11:GT:", self.genome)]
         interval = Interval('chr1', '+', 5, 15, genome37, 11)
         reference_alignment = apply_variants(genome37.dna, variants, interval, reference_alignment=True)[1]
-        self.assertEqual(reference_alignment, [-1, 0, 1, 2, 3, 4, 6, 7, 8, 9])
+        self.assertEqual(reference_alignment, [-1, 0, 1, 2, 3, 4, 7, 8, 9, 10])
```

Comment:
nice. I guess the fact that there was only one index missing instead of 2 (the variant deletes 2 bases) is the tip for the bug?

Reply:
pretty much, happy circumstance in the test case since it was marked as anchored right